main-app/app/scripts/game/services/end-of-game-service.js

```diff
@@ -0,0 +1,33 @@
+(function() {
+    'use strict';
+    angular.module('Tombola.Games.Bingo90.Core')
+        .service('EndOfGame', ['$interval', '$state', function($interval, $state) {
+            var me = this;
+
+            me.checkForWinner = function(response) {
+                if (response.message === "Line") {
+                    me.checkForLine(response);
+                }
+                if (response.message === "Winner") {
+                    me.checkForHouse(response);
+                }
+            };
+
+            me.checkForLine = function(response) {
+                alert('--Line Prize\nWinner: ' + response.payload.winnerInfo.linewinnername + '\nAmount Won: ' + response.payload.winnerInfo.lineprize);
+            };
+
+            me.checkForHouse = function(response) {
+                alert('--House Prize\nWinner: ' + response.payload.winnerInfo.housewinnername + '\nAmount Won: ' + response.payload.winnerInfo.houseprize + '\n--Line Prize\nWinner: ' + response.payload.winnerInfo.linewinnername + '\nAmount Won: ' + response.payload.winnerInfo.lineprize);
+                me.gameEnded();
+            };
+
+            me.gameEnded = function() {
```

Comment:
This is part of the bingo-calling functionality - the fact `me.gameloop` isn't part of this service should perhaps have tipped you off.